src/utils/eventMapper.js

```diff
@@ -0,0 +1,16 @@
+const EVENTS_LIST = ['change', 'keyup', 'keypress', 'keydown', 'focus', 'focusin', 'focusout', 'blur'];
```

Comment:
A może jeszcze event `input`

Reply:
Spoko, jak masz jeszcze jakiś pomysł to dopisze